developer-guide/concepts-and-designs/sig-devops/devops_observability.md

```diff
@@ -0,0 +1,22 @@
+## Expose Pipiline metrics to Prometheus
+
+​	In order to have a more intuitive statistically view about pipelines. we should collect some metrics about pipelines, and then expose an endpoint with metrics where a Prometheus Server can scrape.
+
+​    Now we recommend using  [Jenkins Prometheus Plugin](https://github.com/jenkinsci/prometheus-plugin) to expose metrics about pipelines. Currently only metrics from the [Metrics-plugin](https://github.com/jenkinsci/metrics-plugin) and summary of build duration of jobs and pipeline stages.
```

Comment:
Currently, only...

Reply:
`Currently only metrics from the [Metrics-plugin](https://github.com/jenkinsci/metrics-plugin) and summary of build duration of jobs and pipeline stages. `, this sentence is very strange.